src/labeled_signal.jl

```diff
@@ -158,8 +165,18 @@ data represented by the _signal_.
 If any label span is not entirely contained within the corresponding signal
 span, this will throw an ArgumentError.
 """
-function label_signals(signals, annotations; groups=:recording, kwargs...)
-    labels_table = labels_to_samples_table(annotations; groups, kwargs...)
+function label_signals(signals, annotations;
+                        groups=:recording,
+                        labels_column,
+                        epoch,
+                        encoding,
+                        roundto=nothing)
+    labels_table = labels_to_samples_table(annotations;
+                                            groups,
+                                            labels_column,
+                                            epoch,
+                                            encoding,
+                                            roundto)
```

Comment:
I'd still rather keep this as generic as possible even if we're documenting the kwargs that get passed:
```suggestion
function label_signals(signals, annotations; groups=:recording, kwargs...)
    labels_table = labels_to_samples_table(annotations; groups, kwargs...)
```

This way, if we change the defaults/required kwargs there, we don't have to update here as well, and there's only one place where defaults are defined.

Reply:
won't we have to update the docstring anyway if we change the kwargs?

I think the root of the issue here is the lack of sensible defaults for these kwargs...which is forcing us to document them at the public level yet we want to reserve the right to change them at the private level...

I think to resolve this we could define some defaults then abstract over them in the public function call but call out where to go look if you want to override specific behaviour.